test/oatpp/core/data/mapping/type/StringTest.cpp

```diff
@@ -42,7 +42,6 @@ void StringTest::onRun() {
     oatpp::String s;
     OATPP_ASSERT(!s);
     OATPP_ASSERT(s == nullptr);
-    OATPP_ASSERT(s == (const char*) nullptr);
```

Comment:
It's not a useless test as it seems - there are two different operators: one for `nullptr_t` and one for `const char*`

Reply:
done, with a static cast